### DISCUSSION TITLE
How to use a custom authentication API

### DISCUSSION THREAD

**Discussion by johtso** [blitz-js/blitz#2568] [Q&A]
Is there any documentation on how to integrate an authentication service available via a HTTP API? Would you need to write a custom strategy/adapter of some sort?

I guess ideally you'd want to keep all the nice stuff that the default auth setup does for you, except instead of querying the database to see if a login is valid, it should make an api request. Obviously the same for other user CRUD actions..

Is that something that you can do? Just override the logic for password-check/update user/etc.?

**Comment by flybayer** [comment]
Unfortunately not yet. Are you integrating with a third-party service or a custom API you built yourself?
